reana_db/models.py

```diff
@@ -121,6 +123,15 @@ def get_user_workspace(self):
         """
         return build_workspace_path(self.id_)
 
+    def get_user_disk_usage(self, summarize=True, block_size=None):
+        """Retrieve user disk usage information."""
+        try:
+            workspace_path = self.get_user_workspace()
+            disk_usage = get_disk_usage(workspace_path, summarize, block_size)
+            return disk_usage[0]["size"]
```

Comment:
There is just a little detail that needs to be amended: if someone passes a different value than the default to `summarize`, `False`, they will still be receiving just the first element instead of everything... we have two options:
1. Remove the `summarize` parameter and hardcode its value to `True` inside the function. Assuming that for users we want just the overall usage
2. Have some logic which will change the return value depending on the value of `summarize`

In my opinion (1.) makes more sense as I see no use case in which we want a detailed `du` for all workflows of a user (potentially hundreds of workflows times the number of files they contain in their workspace...), what do you think?

Reply:
Option (1) makes more sense to me as well. I've updated the code to accordingly